packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 4.2.1
+
+* [java] Fix 'Method invocation 'get' may produce 'NullPointerException''
+* [java] Fix 'Call to 'asList()' with only one argument'
+* [java] Fix 'Casting 'args.get(...)' to 'Object' is redundant'
```

Comment:
Make sure you check out the style guide for plugins/packages. https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style